cg/meta/upload/coverage.py

```diff
@@ -19,9 +19,9 @@ def __init__(self, status_api: Store, hk_api: HousekeeperAPI, chanjo_api: Chanjo
 
     def data(self, analysis_obj: Analysis) -> dict:
         """Get data for uploading coverage."""
-        family_id = analysis_obj.family.internal_id
-        data = {"family": family_id, "family_name": analysis_obj.family.name, "samples": []}
-        for link_obj in analysis_obj.family.links:
+        family_id = analysis_obj.case.internal_id
```

Comment:
```suggestion
        case_id = analysis_obj.case.internal_id
```

Reply:
I'm not going to touch any variable names in this PR, but it should be fixed. I think I'll just introduce bugs if I do this here